WordPress/Classes/ViewRelated/Activity/JetpackActivityLogViewController.swift

```diff
@@ -32,4 +37,20 @@ class JetpackActivityLogViewController: BaseActivityListViewController {
 
         WPAnalytics.track(.activityLogViewed)
     }
+
+    private func configureBanner() {
+        containerStackView.addArrangedSubview(jetpackBannerView)
+
+        NSLayoutConstraint.activate([
+            jetpackBannerView.heightAnchor.constraint(equalToConstant: 50)
```

Comment:
We just added this [static property](https://github.com/wordpress-mobile/WordPress-iOS/blob/1e59f3941b7964d87bb8b8b9d3e7a517cdcd0f72/WordPress/Classes/ViewRelated/Jetpack/Branding/Banner/JetpackBannerView.swift#L27) yesterday for this value - could we use it here instead of the literal?

Reply:
👍

Reply:
I updated the constraint to keep it consistent. I also used the static property.